client/src/pages/BasketPage/BasketPage.js

```diff
@@ -1,27 +1,79 @@
 import React, { Component } from "react";
-import { MoneyButton, MealCard } from "../../components";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+import removeBasketItem from "../../actions/removeBasketItem";
+import { MealCard } from "../../components";
 import { MealName, Delivery, OrderInfo, Cost } from "./BasketPage.style";
 
-import mealImage from "../../assets/meal_image.png";
-
-export default class MealListPage extends Component {
+class BasketPage extends Component {
+  basketIsEmpty = () => Object.keys(this.props.basket).length === 0;
   render() {
     return (
       <React.Fragment>
-        <h1>Your Basket</h1>
-        <MealCard mealImage={mealImage}>
-          <MealName>Russian tomato sauce with shashlikee</MealName>
-          <Delivery>For delivery for 7pm on 22/09/18</Delivery>
-          <OrderInfo>
-            <Cost>Cost: £5.00</Cost>
-            <select>
-              <option value="">1 Portion</option>
-            </select>
-            <button>Delete</button>
-          </OrderInfo>
-        </MealCard>
-        <MoneyButton />
+        {this.basketIsEmpty() ? (
```

Comment:
I think this deserves more than the if shorthand. It wilkl more readable if you did something like:
if(this.basketIsEmpty()) 
return(x) 
else 
return(y)